starter_code/express-spotify-app/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require("express");
+const hbs = require("hbs");
+var SpotifyWebApi = require("spotify-web-api-node");
+const app = express();
+const path = require('path')
+app.set("view engine", "hbs");
+app.set("views", __dirname + "/views");
+app.use(express.static(path.join(__dirname, 'public')));
+hbs.registerPartials(__dirname + "/views/partials");
+
+var clientId = "47588c328876436cbe7bb158c10dfebc",
```

Comment:
**This are sensitive data**, so I recommend always save as a environment variable. We already learn this, so in your future project always add this kind of data into your `.env` file, and make sure that your `.env` file is mentioned in the `.gitignore` file before to make any commit :grin: 

Then I will told you a funny not so funny story about this :wink: